mysql-test/main/func_misc.test

```diff
@@ -18,12 +18,12 @@ select hex(inet_aton('127.1'));
 select hex(inet_aton('127.1.1'));
 
 select length(uuid()), charset(uuid()), length(unhex(replace(uuid(),_utf8'-',_utf8'')));
+select length(uuidv7()), charset(uuidv7()), length(unhex(replace(uuidv7(),_utf8'-',_utf8'')));
```

Comment:
One of the key properties of [uuid7](https://uuid7.com/) is time-sortability. 

> UUIDv7 values are time-sortable, which means you can sort them in increasing order based on when they were generated. This makes time-based queries more efficient and intuitive.

- Does this test verify that property of the implementation of the `uuid7` function? 
- Does it do it in as simple and direct a way possible?

I think the answer to both of these is **no**. An incorrect implementation of `uuid7`  implementation that [always returns a fixed hexadecimal string](http://bilbo.hobbiton.org/dilbert/browse.php?y=2016&m=04&d=01) would equally well pass this test, wouldn't it?

Reply:
Consider adding something (here or in another test) that actually verifies the functional properties of the `uuid7` implementation, including key properties like time-sortability.

Reply:
Hello Dan, look at the changes in the file `plugin/type_uuid/mysql-test/type_uuid/order.test`. This test has been suggested by Grooverdan and it tests exactly that generated UUIDv7s are time-sortable and unique. In my opinion that test should be enough and no better way of testing this property comes to my mind, but, if you have suggestions I am very open to writing more/better tests :)  

I agree that the line that you commented on is incomplete and redundant with the other test, do you think it would be better to remove it?

Reply:
> look at the changes in the file plugin/type_uuid/mysql-test/type_uuid/order.test. This test has been suggested by Grooverdan and it tests exactly that generated UUIDv7s are time-sortable and unique.   In my opinion that test should be enough and no better way of testing this property comes to my mind, but, if you have suggestions I am very open to writing more/better tests :)

Thanks. My apologies for overlooking this in my initial review. I agree, this test is great, and very clear.

> Hello Dan, look at the changes in the file plugin/type_uuid/mysql-test/type_uuid/order.test. This test has been suggested by Grooverdan and it tests exactly that generated UUIDv7s are time-sortable and unique. In my opinion that test should be enough and no better way of testing this property comes to my mind, but, if you have suggestions I am very open to writing more/better tests :)

> I agree that the test that you commented on is incomplete and redundant with the other test, do you think it would be better to remove it?

Yes, completely agreed.  `func_misc.test` mostly checks some superficial properties of a laundry list of functions, and I don't think it's likely to catch future regressions or easy to understand in this format.

So I think it's unnecessary to add to it here.

Reply:
Looking good to me in https://github.com/MariaDB/server/pull/3093/commits/46bef2ac01750635ea81a733d7664c853154c8ae :rocket: :+1: